python/lsst/pipe/tasks/selectImages.py

```diff
@@ -590,3 +591,119 @@ def run(self, wcsList, bboxList, coordList, psfList, dataIds, **kwargs):
                       f"range of {psfSizes[indices.index(output[0])]}"
                       f"--{psfSizes[indices.index(output[-1])]} arcseconds")
         return output
+
+
+class BestSeeingSelectVisitsConnections(pipeBase.PipelineTaskConnections,
+                                        dimensions=("tract", "patch", "skymap", "band", "instrument"),
+                                        defaultTemplates={"coaddName": "bestSeeing"}):
+    sourceTables = pipeBase.connectionTypes.Input(
+        doc="Source table with PSF size information",
```

Comment:
Can you help me understand where this input will usually come from? Is it just the `src` produced by single-frame processing, or something fancier (since it needs PSF size info as well as visit info)? I feel like the butler needs to be told where to get this from *somehow* to run your new pipeline, but I don't see where that happens.